src/mapgen.cpp

```diff
@@ -5827,7 +5918,7 @@ void map::draw_lab( mapgendata &dat )
         } // end aboveground vs belowground
 
         // Ants will totally wreck up the place
-        if( is_ot_match( "ants", terrain_type, ot_match_type::contains ) ) {
+        if( true ) { //###is_ot_match( "ants", terrain_type, ot_match_type::contains ) ) {
```

Comment:
I guess this should be removed.

Reply:
Crap! You're correct. I did do that to be able to create trashed facilities, as debug spawning them didn't seem to work.